src/main/java/com/zendesk/maxwell/MaxwellFilter.java

```diff
@@ -137,7 +137,7 @@ public boolean isTableBlacklisted(String databaseName, String tableName) {
 	}
 
 	public static boolean isSystemBlacklisted(String databaseName, String tableName) {
-		return "mysql".equals(databaseName) && "ha_health_check".equals(tableName);
+		return "mysql".equals(databaseName) && ("ha_health_check".equals(tableName) || "rds_heartbeat2".equals(tableName));
```

Comment:
Also, probably worth adding this to `testSystemBlacklist` in `src/test/java/com/zendesk/maxwell/MaxwellIntegrationTest.java`